spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -471,6 +471,32 @@ dataflow:>task list
 ╠══════════════════════════╪═══════════════════════════════════════════════════════════════
 ╚══════════════════════════╧═══════════════════════════════════════════════════════════════
 ```
+==== Stopping a Composed Task
+In cases where a composed task execution needs to be stopped.  This can be done
+via the:
+
+* RESTful API
+* Spring Cloud Data Flow Dashboard by selecting the Job's tab and then
+clicking the stop button by the job execution that needs to be stopped.
```

Comment:
Not at this time.  Added story https://github.com/spring-cloud/spring-cloud-dataflow/issues/1362